plugins/pulseaudio/PulseAudioDevice.h

```diff
@@ -45,10 +45,15 @@ class AUD_PLUGIN_API PulseAudioDevice : public SoftwareDevice
 	class PulseAudioSynchronizer : public DefaultSynchronizer
 	{
 		PulseAudioDevice* m_device;
+		pa_usec_t m_time_start = 0;
+		double m_seek_pos = 0.0f;
+		double m_timeline_pos = 0.0f;
 
 	public:
 		PulseAudioSynchronizer(PulseAudioDevice* device);
 
+		void update_time_start();
```

Comment:
Please follow the camelCase naming scheme that audaspace uses for methods. Since you should implement `play()`, this would be renamed anyway.